lib/gs-plugin-loader.c

```diff
@@ -3811,6 +3811,9 @@ gs_plugin_loader_job_timeout_cb (gpointer user_data)
 {
 	GsPluginLoaderHelper *helper = (GsPluginLoaderHelper *) user_data;
 
+	g_debug ("was cancelling job as it took too long, not anymore");
+	return G_SOURCE_REMOVE;
```

Comment:
I think `helper->timeout_id` should still be set to 0 in this case.

Reply:
Good catch, updated.